design.py

```diff
@@ -1,16 +1,40 @@
 import sys
+import json
 #prompt a user to create a recipe
 #Ask for Ingredients
 #Ask for Instructions
 #Save/Store it into dictionary
 def create():
-    ask_recipe = input("Recipe Name: ")
-    ask_instruction = input("Instructions: ")
-    ask_create = input("Ingredients: ")  
+    ask_recipe = ''
+    ask_instruction = ''
+    ask_ingredients = ''
+    recipe_dict = dict()
+    #counter = str(1)
+    print("Create a recipe! Type 'exit' to leave")
+    for ask_recipe in range (1):
+        ask_recipe = input("Recipe Name: ")
+        recipe_dict.update(ask_recipe = 'Nothing Yet')
+        if ask_recipe == 'exit':
+            exit(1)
 
+    while ask_instruction != 'exit':
+        ask_instruction = input("Instructions: ")
+        if ask_instruction == 'exit':
```

Comment:
There is no need to nest these, it only makes things weird which is probably why you're having issues. 

Remember, before we ever get to put ANYTHING in the dictionary we need 3 things. We need recipe_name, ingredients, and instructions.

In the design doc we know what data types we're using for this.

recipe name = string
ingredients = list
instructions = list 

So now that we know that, we just have to collect those 3 things and then we can worry about how to store them in a dictionary later.

Instead what you should do here is very simple.

Step 1: Ask user for the recipe name, then store it in a variable.
Step 2: Create a for loop where we can ask the user for ingrediants until he/she is finished. Append these to a list.
Step 3: Create a for loop where we can ask the user for instructions until he/she is finished. Append these to a list.